authui/packages/vite-plugin-authgear-authui/src/index.js

```diff
@@ -280,8 +273,8 @@ function buildPlugin({ input }) {
 
           for (const node of head.childNodes) {
             if (node.tagName === "LINK") {
-              const hashedName = removeLeadingSlash(node.getAttribute("href"));
-              const key = nameWithoutHash(hashedName);
+              const href = node.getAttribute("href");
+              const key = nameWithoutHash(path.basename(href));
```

Comment:
path.basename only keep the last part of the path, e.g. `/img/icon-abcd.svg` -> `icon-abcd.svg`, not sure if this is expected.

Reply:
For existing usage pattern, yes, it is expected. It is because the existing usage of `GeneratedStaticAssetURL` expects basename only.

Reply:
🤔 Then maybe there will be some problem if there are multiple files with the same name in different directories?

Reply:
So there is indeed some problem if there are multiple files with the same filename in different directories, because it causes key collision in the manifest json.
It works for now as we don't have files with the same name, not sure if it would be a concern in the future. Maybe we can ignore it for now.

Reply:
We should migrate to use `manifest: true` and `.vite/manifest.json` instead of writing our own `manifest.json`. But that's more complicated so I leave it for now.